app/controllers/uploads_controller.rb

```diff
@@ -23,10 +23,10 @@ def destroy
     @upload = Upload.find(params[:id])
     @upload.destroy
     if current_user.admin?
-      flash[:danger] = "File deleted."      
+      flash[:danger] = "File deleted."
       redirect_to admin_dashboard_path
     else  
-      flash[:danger] = "File deleted."            
+      flash[:danger] = "File deleted."
```

Comment:
@cmacaulay @AELSchauer the second flash message is for the redirect going to the home_path.  Tho i believe only one is necessary, now that Ive looked it over some more.